bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/config/rm-config.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+	https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:broadlink:rmconfig">
+		<parameter name="ipAddress" type="text">
+			<context>network-address</context>
+			<label>Network Address</label>
+			<description>Network address of the device.</description>
+			<required>true</required>
+		</parameter>
+		<parameter name="staticIp" type="boolean">
+			<context>static-ip</context>
+			<label>Static IP</label>
+			<description>Will the device always be given this network address?</description>
+			<required>true</required>
+			<advanced>true</advanced>
+			<default>true</default>
+		</parameter>
+		<parameter name="port" type="integer">
+			<default>80</default>
+			<context>network-port</context>
+			<label>Network Port</label>
+			<description>Network port of the device.</description>
+			<required>true</required>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="macAddress" type="text">
+			<context>mac-address</context>
+			<label>MAC Address</label>
+			<description>MAC address of the device.</description>
+			<required>true</required>
+		</parameter>
+		<parameter name="pollingInterval" type="integer" min="30" step="1">
+			<label>Polling Interval</label>
+			<description>The interval in seconds for polling the status of the device.</description>
+			<required>true</required>
+			<default>30</default>
+			<advanced>true</advanced>
+		</parameter>
+		<parameter name="mapFilename" type="text">
+			<label>Map File</label>
+			<description>Enter name of file containing mapping of commands to IR codes</description>
+			<default>broadlink.map</default>
+			<required>true</required>
+		</parameter>
+		<parameter name="retries" type="integer" min="0" max="1">
```

Comment:
Why is this min 0 and max 1? is it a valid case someone wants 3 retries? I do not have the hardware nor have I looked at the API so I do not know if this makes sense or not.

Reply:
The `retries` thing was overly-complicated - I've removed it